apps/src/templates/manageStudents/ShowSecret.jsx

```diff
@@ -111,8 +111,8 @@ class ShowSecret extends Component {
   };
 
   render() {
-    const {resetDisabled} = this.props;
-    const tooltipId = resetDisabled && _.uniqueId();
```

Comment:
You should probably remove this prop from parent components (only `ManageStudentsTable` in this case).

Reply:
Yep, I had missed this. Also will be updating an associated unit test.

Reply:
✅